domain/secretbackend/state/types.go

```diff
@@ -188,3 +189,51 @@ type ModelCloudCredentialRow struct {
 	// OwnerName is the name of the credential owner.
 	OwnerName string `db:"owner_name"`
 }
+
+// CloudRow represents a single row from the state database's cloud table.
+type CloudRow struct {
```

Comment:
Why are we redefining CloudRow here but for credential are exporting it and reusing?

Reply:
Because they are for different. This is a more simpler with some different fields due to the different SQL.